.github/workflows/weekly_ci.yml

```diff
@@ -8,32 +8,11 @@ env:
   BRANCH_NAME: "weekly-ci"
   COMMIT_MESSAGE: "Weekly CI check"
   COMMENT: "[citest]"
+  MAIN_BRANCH: "main"
```

Comment:
please don't hardcode the main branch name - we will be changing the remaining `master` branches to `main` at some point in the near future - instead, please use this method for determining the main branch name - https://github.com/linux-system-roles/auto-maintenance/blob/master/role-make-version-changelog.sh#L42

Reply:
Thank you, I added the function to the script